config/config.yaml

```diff
@@ -81,15 +67,14 @@ node-groups:
       nginx.ingress.kubernetes.io/session-cookie-name: "SWARMGATEWAY"
       nginx.ingress.kubernetes.io/session-cookie-path: "default"
       nginx.ingress.kubernetes.io/ssl-redirect: "true"
-    ingress-class: "nginx-internal"
+      nginx.ingress.kubernetes.io/whitelist-source-range: "142.132.221.245/32" #VPN IP, or your IP (but ethswarm.org is needed)
```

Comment:
Not needed now, vpn is whitelisted globally

Reply:
Still getting the `403 Forbidden`if I do not whitelist... Does it depend on VPN config?

Reply:
No, its intended to resolve testnet.internal... maybe you still have a 'public' ingress

Reply:
@darkobas2 whitelist ingress anotation is removed. After the changes you made, I have tested configuration and it works without it.